Source/WebCore/page/scrolling/mac/ScrollerPairMac.h

```diff
@@ -134,8 +134,8 @@ class ScrollerPairMac : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<S
 
     ScrollbarHoverState m_scrollbarHoverState;
 
-    ScrollerMac m_verticalScroller;
-    ScrollerMac m_horizontalScroller;
+    const UniqueRef<ScrollerMac> m_verticalScroller;
+    const UniqueRef<ScrollerMac> m_horizontalScroller;
```

Comment:
Now that you made these `const` we should be able to get rid of the checked methods above.

Reply:
Cleaning up in https://github.com/WebKit/WebKit/pull/53359.